Front/js/scripts.js

```diff
@@ -224,59 +214,29 @@ function getTemplateByID(num) {
         },
         error: function () {
             console.log('Error - get -  TemplatesBYID');
-            // top.location.href="404.html";
         }
     });
 }
 
 
-function DeleteByID(num) {
-    console.log(num);
-    let id = localStorage.getItem(num);
-    let id2 = JSON.stringify(id);
-    let id3 = JSON.parse(id2);
-    let id4 = id.replace('"', '');
-    let id5 = id4.replace('"', '');
-    let data = {
-        "template_id": id5
-    }
-
-    console.log("the id is", id5);
-
 
-    $.ajax({
-        url: 'http://localhost:3000/template',
-        type: 'DELETE',
-        data: (JSON.stringify(data)),
-        success: function (template) {
-            console.log(template);
-            console.log(template.name);
-            console.log(template.html);
-            let html = template.html;
-            document.getElementById("email_message").innerHTML = html;
-            location.reload();
-        },
-        error: function (error) {
-            console.log(error);
-        }
+function clickTemplate() {
+    $(document).on('click', '#template', function (e) {
+        e.preventDefault();
+        alert("herrre");
     });
 }
 
-function createTemplateTable(nameArray, IdArray, htmlArray) {
-    var mytable = "<table class='table table-striped'> <thead> <tr> <th scope='row'>NUM</th> <th scope='row'>ID</th> <th scope='row'>Name</th><th scope='row'>Body</th></tr></thead><tbody>";
-    console.log(nameArray.length);
-    for (let i = 0; i < nameArray.length; i++) {
-        mytable += "<tr>";
 
-        mytable += "<th scope='row'>" +
-            "<button onclick='getTemplateByID(" + `${[i]}` + ")'" + "class=button>Add Template</button>" +
-            "<button onclick='DeleteByID(" + `${[i]}` + ")'" + "class=button>Delete Template</button>" +
-            "<button onclick='edit()'" + "class=button>Edit Template</button>"
-        {/* mytable += "<th scope='row'><div id=list-items><a href=#> Hindi </a><a href=#> English </a><a href=#> Spanish </a> <a href=#> Chinese </a> <a href=#> Japanese </a> </div><button id="+ `${i}`+">" + "</th>"; */
-        }
+function createTemplateTable(nameArray,IdArray,htmlArray){
```

Comment:
Not sure if those functions are needed anymore since we have new dynamicly loaded scripts in templatesScript.js